tests/zero_code_change/test_pytorch_disengage_smdebug.py

```diff
@@ -0,0 +1,53 @@
+# Third Party
+# Standard Library
+from unittest.mock import patch
+
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from tests.zero_code_change.pt_utils import Net, get_dataloaders
+
+# First Party
+import smdebug.pytorch as smd
+from smdebug.core.utils import SagemakerSimulator
+
+
+@patch("smdebug.core.config_validator.is_framework_version_supported", return_value=False)
```

Comment:
Can we use the same decorator for the other tests? This is much cleaner.

Reply:
Done.